xdsl/ir.py

```diff
@@ -503,7 +503,11 @@ def verify(self, verify_nested_ops: bool = True) -> None:
         self.verify_()
         if verify_nested_ops:
             for region in self.regions:
-                region.verify()
+                try:
+                    region.verify()
+                except:
+                    print(f'Verification error for operation `{self}`')
```

Comment:
```                    print(f'Verification error for operation `{self}`' ... for (print specific region???? for more info?))```

Reply:
Not sure what you mean :)

Reply:
WOuld it be of any use to add the region name in the error?

Reply:
My experience so far was that the Op was already super useful, as it's usually the most specific verification point that I had customised. I then set a breakpoint in the Op and found the bug. Let's add the region when we've found a case that it's been helpful in debugging?

Reply:
SHould we try to think the other way round? Can we write a test that makes this info useful? Can you think of such a case?